src/Sylius/Bundle/LocaleBundle/Templating/Helper/LocaleHelperInterface.php

```diff
@@ -13,13 +13,26 @@
 
 namespace Sylius\Bundle\LocaleBundle\Templating\Helper;
 
-use Symfony\Component\Templating\Helper\HelperInterface;
-
-interface LocaleHelperInterface extends HelperInterface
+interface LocaleHelperInterface
 {
     /**
      * @param string $code The code to be converted to a name
      * @param string|null $localeCode The locale that the returned name should be in
      */
     public function convertCodeToName(string $code, ?string $localeCode = null): ?string;
+
+    /**
+     * Returns the canonical name of this helper.
+     */
+    public function getName(): string;
```

Comment:
Templating is no more available on SF7. Even if this is possible to still using it, I prefer to remove this dependency, cause it's deprecated and will be no more maintained.